Sources/WebUI/WebView.swift

```diff
@@ -106,6 +106,13 @@ public struct WebView {
         webView.allowsBackForwardNavigationGestures = allowsBackForwardNavigationGestures
         webView.allowsLinkPreview = allowsLinkPreview
         webView.isRefreshable = isRefreshable
+    }
+
+    /// Loads the argument URL received during WebView initialization only once.
+    ///
+    /// This must be used after applyModifiers.
```

Comment:
Documentation for the method is not necessary because it is internal.

Reply:
Remove the documentation. [3b9ccbe](https://github.com/cybozu/WebUI/pull/1/commits/3b9ccbe2975a716764befea63f3a808e4783813c)